helm/values.yaml

```diff
@@ -125,6 +125,10 @@ corda:
       identityManagerAddress: ""
       # networkmapAddress defines the accesspoint for the Network Map server (public domain name+port)
       networkmapAddress: ""
+      # compatibilityZoneEnabled set to true for testnet
```

Comment:
compatibilityZoneEnabled defines if you are connecting to a compatibility zone rather than using the above identityManagerAddress+networkmapAddress. Set this to enabled to be able to join Corda Testnet along with compatibilityZoneURL.